compiler/pash_ptempfile_name.sh

```diff
@@ -2,13 +2,5 @@
 
 distro=${1??Distro not given}
 # now do different things depending on distro
-case "$distro" in
-    freebsd*)  
-        # bsd is so fun :)
-        tmp=$(TMPDIR=$PASH_TMP_PREFIX mktemp -t pash_XXXXXXXXXX)
-        echo "${tmp}"
-        ;;
-    *)
-        mktemp --tmpdir="$PASH_TMP_PREFIX" -u pash_XXXXXXXXXX
-        ;;
-esac
+tmp_file=$(TMPDIR=$PASH_TMP_PREFIX mktemp -t pash_XXXXXXXXXX)
+echo "${tmp_file}"
```

Comment:
Also this is redundant, we can simply run the above command and its output will be send to stdout. Now we capture the commands output in a variable and then send it to output through stdout :)